content/encrypt-extra.md

```diff
@@ -18,17 +18,17 @@ tableOfContents: true
 
 The <u>Disks</u> application can be used to encrypt the extra drive and it comes pre-installed on Pop!_OS and Ubuntu.
 
-In this screenshot the extra drive can be seen and then select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Volume' window.
+In this screenshot, the extra drive can be seen. Select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Partition' dialog.
 
 ![Disks](/images/encrypt-extra/Disk-Menu.png)
 
-Then use the 'Format Volume' window to format the volume with the options to name the volume and use the Ext4 partition with LUKS.
+When the 'Format Volume' dialog window opens, set the volume name and select the `Ext4` type with `LUKS`. Click on `Next` at the top right of the dialog, then click on format.
```

Comment:
Since naming the volume is optional, do you think it'd be better to reword that so it doesn't sound like the user has to give it a name?

Reply:
Yea, sure